src/main.rs

```diff
@@ -200,7 +204,7 @@ fn init_log() {
 }
 
 #[cfg(not(feature = "logging"))]
-fn init_log() {}
+const fn init_log() {}
```

Comment:
This makes the signature incompatible with the other `init_log` (when logging is enabled). Please revert